src/applications/model/gaming-streaming-server.h

```diff
@@ -23,16 +23,22 @@
 #ifndef GAMING_STREAMING_SERVER_H
 #define GAMING_STREAMING_SERVER_H
 
+#include "ns3/socket.h"
```

Comment:
is it needed? you only have a `Ptr<Socket>` thus a forward declaration should be enough

Reply:
I should either declare it or include it.

Reply:
yes, i'm saying that it's better to declare, the compiler shouldn't complain